include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid
+   *
+   * Following alignment conditions have to be fulfilled: MB_A = NB_B and NB_A = MB_B
```

Comment:
in class description we have

> ...into $MB$ by $NB$ blocks...

not sure how it renders, but probably you should do $MB_A$ and alike here and below in this PR to keep it consistent.

Reply:
full stop

Reply:
I will change it at the one point in the class description, if that is fine with you.

Reply:
i would actually prefer the opposite, i think it will render better as latex formula. Otherwise you have it as a text `MB_A`

Reply:
p.s. that's also how we do it all other places inside ScaLAPACK, including the most recent PR with some docu changes https://github.com/dealii/dealii/pull/5875/

Reply:
Right ...